### ISSUE TITLE
add ability to specify a stream

### ISSUE THREAD

**Issue by andy5995** [dafky2000/simplectemplate#54]
Using render_template_file prints the output to stdout. Can a desired stream be passed so the output gets written directly to a file?

**Comment by dafky2000** [comment]
Good idea! Not a priority right now but perhaps I could add an option to the `RenderOptions` structure for this.
